pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md

```diff
@@ -70,298 +71,510 @@ us-west1
+      value = "${module.dcos.public-agents-loadbalancer}"
+    }
+    ```
 
-output "public-agents-loadbalancer" {
-  value = "${module.dcos.public-agents-loadbalancer}"
-}
-```
+2) Re-run the new execution plan with the changes you have made to the `main.tf`configuration file.  
 
-2) The changes are made to `main.tf`, now re-run the new execution plan.  
+    ```bash
+    terraform plan -out=plan.out
+    ```
 
-```bash
-terraform plan -out=plan.out
-```
+    This step ensures that the state is stable and confirms that you can create the resources necessary to scale the private agents to the desired number. Executing the plan adds the following resources as a result of scaling up the cluster’s private agents:
+    - One instance resource
+    - Two null resources which handle the DC/OS installation and prerequisites in the background.
 
-This step ensures that the state is stable and to confirm that you will create the resources necessary to scale the private agents to the desired number.
+    You should see a message similar to the following:
+<p align=center><img src="./images/scale/terraform-plan.png" /></p>
 
-<p align=center>
-<img src="./images/scale/terraform-plan.png" />
-</p>
+3) Run the following command to have Terraform deploy the new set of resources:
 
-After executing the plan, you should see a message similar to above message. There will be `three` resources added as a result of scaling up the cluster’s private agents (1 instance resource & 2 null resources which handle the DC/OS installation and prerequisites in the background).
+    ```bash
+    terraform apply plan.out
+    ```
 
-3) Now, that the plan is set, get Terraform to build/deploy the new set of resources.
+    You should see an output similar to the following:
+  <p align=center><img src="./images/scale/terraform-apply.png" /></p>
+  
+4) Check your DC/OS cluster using the DC/OS UI to verify the additional agents have been added. 
 
-```bash
-terraform apply plan.out
-```
+    You should see `four` total nodes connected. For example:
+  <p align=center><img src="./images/scale/node-count-4.png" /></p>
 
-<p align=center>
-<img src="./images/scale/terraform-apply.png" />
-</p>
+# Upgrading the cluster
+Terraform also makes it easy to upgrade the DC/OS cluster to a newer version of DC/OS.
 
-Once you see an output like the message above image, then check your DC/OS cluster to ensure the additional agents have been added.  
+Read more about the upgrade procedure that Terraform performs in the official [DC/OS Upgrade](https://docs.mesosphere.com/1.10/installing/production/upgrading/) documentation.
 
-Now, you should see `four` total nodes connected like below via the DC/OS UI.
+To perform an upgrade:
 
-<p align=center>
-<img src="./images/scale/node-count-4.png" />
-</p>
+1) Open the `main.tf` configuration file.
 
+2) Modify the current DC/OS Version (`dcos_version`) to `1.11.5` and set the `dcos_install_mode` parameter to `upgrade`. 
```

Comment:
>Change the current DC/OS version (`dcos_version`) variable to `1.12.0` and set the `dcos_install_mode` variable to `upgrade`.

By default, the `dcos_install_mode` variable is set to `install` in order to enable creation and scalability of your initial DC/OS cluster.  In order to upgrade with the Universal Installer, you must explicitly set the `dcos_install_mode` varibale to `upgrade`.